applications/asset_tracker/src/light_sensor/light_sensor.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <spinlock.h>
+#include "light_sensor.h"
+
+#define LIGHT_SENSOR_DEV_NAME "BH1749"
+
+typedef enum {
+	LS_CH__BEGIN = 0,
+
+	LS_CH_RED = LS_CH__BEGIN,
+	LS_CH_GREEN,
+	LS_CH_BLUE,
+	LS_CH_IR,
+
+	LS_CH__END
+} ls_ch_type_t;
+
+struct ls_ch_data {
+	enum sensor_channel type;
+	struct sensor_value data;
+};
+
+static struct ls_ch_data ls_ch_red = { .type = SENSOR_CHAN_RED };
+static struct ls_ch_data ls_ch_green = { .type = SENSOR_CHAN_GREEN };
+static struct ls_ch_data ls_ch_blue = { .type = SENSOR_CHAN_BLUE };
+static struct ls_ch_data ls_ch_ir = { .type = SENSOR_CHAN_IR };
+
+static u8_t *ls_dev_name = LIGHT_SENSOR_DEV_NAME;
+static struct device *ls_dev;
+static struct k_spinlock ls_lock;
+static struct ls_ch_data *ls_data[LS_CH__END] = { [LS_CH_RED] = &ls_ch_red,
+						  [LS_CH_GREEN] = &ls_ch_green,
+						  [LS_CH_BLUE] = &ls_ch_blue,
+						  [LS_CH_IR] = &ls_ch_ir };
```

Comment:
Two lines can be removed

Reply:
i removed them.  they are still showing on mobile though.